components/brave_shields/browser/cookie_pref_service_browsertest.cc

```diff
@@ -4,6 +4,7 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "brave/components/brave_shields/browser/brave_shields_util.h"
+#include "chrome/browser/content_settings/host_content_settings_map_factory.h"
```

Comment:
Shouldn't we move the browsertest to `brave/browser`?

Reply:
Moved to brave/browser/brave_shields/...